tests/datasets/test_datasets.py

```diff
@@ -29,3 +33,18 @@ def test_load_dataset(name):
         assert df_train.shape[1] == df_test.shape[1]
         if y_test is not None:
             assert y_test.shape[0] == df_test.shape[0]
+
+    shutil.rmtree(tempdir)
+
+
+def test_load_public_path():
+    tempdir = tempfile.mkdtemp()
+    path = _load_public_path(
+        'https://github.com/gfournier/aikit-datasets/releases/download/titanic-1.0.0/titanic.tar.gz',
+        cache_dir=tempdir,
+        cache_subdir='datasets')
+    assert path == os.path.join(tempdir, 'datasets', 'titanic.csv')
+    os.unlink(os.path.join(tempdir, 'datasets', 'titanic.csv'))
+    os.unlink(os.path.join(tempdir, 'datasets', 'titanic.tar.gz'))
+    os.rmdir(os.path.join(tempdir, 'datasets'))
```

Comment:
this fails because dataset not empty, indeed there is a 
"._titanic.csv" in the archive that is not delete.
don't know what that file is

Reply:
Should be ok now, uploaded a new archive.